src/tsickle.ts

```diff
@@ -61,7 +62,7 @@ export function mergeEmitResults(emitResults: EmitResult[]): EmitResult {
   for (const er of emitResults) {
     diagnostics.push(...er.diagnostics);
     emitSkipped = emitSkipped || er.emitSkipped;
-    emittedFiles.push(...er.emittedFiles);
+    emittedFiles.push(...(er.emittedFiles as string[]));
```

Comment:
I vaguely recall Rado has a patch for this, do you know why it fails?  I think we concluded to put a `!` on it instead of the cast.

Reply:
it failed because the type of `er.emittedFiles` is `unknown`. Happy to take some other fix

Reply:
changed to non-null assertion operator, it works too

Reply:
see https://github.com/angular/tsickle/pull/990

Reply:
Let's get this in first, and I will do the "proper fix" with #990 afterwards.

Reply:
I went ahead and applied your same change here too rather than check in an intermediate state